### PR TITLE
Installation in Readme

### PR DESCRIPTION
**Pull request by cemyld**
PR adds a guide on how to install the phageParser with using virtual environments for future collaboration work.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,11 +54,38 @@ More research is needed to better understand how bacteria use their CRISPR syste
 
 ## Installation
 
-This package depends on Biopython, which can be installed as follows:
+You can download the source code of the project by git:
+`git clone git@github.com:phageParser/phageParser.git`
```
Comment (mbonsma) [inline README.md]: Replace this with the https link I think, most people use https and not SSH.

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Many bacterial and archaeal genomes have been sequenced, and [a large fraction o
 * Spacer self-matches to host genome
 * *cas* gene content and inferred CRISPR type
 
-We will collect these annotations in a database that can users can query through a GUI (graphical user interface). Neither of these exist yet, and we are looking for contributors! 
+We will collect these annotations in a database that can users can query through a GUI (graphical user interface). Neither of these exist yet, and we are looking for contributors!
```
Comment (mbonsma) [inline README.md]: It looks like something with your text editor is changing line endings? I'm not sure why all these other paragraphs are showing up as edited.

Comment (cemyld) [inline README.md]: I think the editor was removing the trailing whitespaces automatically, I suggest we keep it unless it's disrupting the markdown format.

File: README.md (hunk 1 above)
Comment (cemyld) [inline README.md]: Yeah, changed it to https

**Pull request merged**